src/kitsuneTools.ts

```diff
@@ -17,12 +26,37 @@ export class KitsuneTools {
         let sig0 = utils.splitSignature(appointment.stateUpdate.signatures[0]);
         let sig1 = utils.splitSignature(appointment.stateUpdate.signatures[1]);
 
-        const tx = await contract.setstate(
-            [sig0.v - 27, sig0.r, sig0.s, sig1.v - 27, sig1.r, sig1.s],
-            appointment.stateUpdate.round,
-            appointment.stateUpdate.hashState
-        );
-        return await tx.wait();
+        try {
+            logger.debug("a1");
+
+            let trying = true;
+            let tries = 0;
+            let tx;
+            while (trying && tries < 10) {
+                try {
+                    tx = await contract.setstate(
+                        [sig0.v - 27, sig0.r, sig0.s, sig1.v - 27, sig1.r, sig1.s],
```

Comment:
I thought we add 27 to v .....

Reply:
The contract expects 0 or 1. It then adds 27.

splitsig creates 27 or 28